frontend/packages/console-app/src/components/cloud-shell/CloudShellExec.tsx

```diff
@@ -142,27 +177,63 @@ const CloudShellExec: React.FC<CloudShellExecProps> = ({
         );
     }
 
+    setWsReopening(false);
+
     return () => {
       unmounted = true;
       websocket.destroy();
     };
-  }, [tick, container, flags, impersonate, namespace, podname, shcommand, t]);
+  }, [
+    tick,
+    container,
+    flags,
+    impersonate,
+    namespace,
+    podname,
+    shcommand,
+    t,
+    workspaceName,
+    wsReopening,
+  ]);
 
   if (wsError) {
     return (
-      <LoadError
-        message={wsError}
-        label={t('cloudshell~OpenShift command line terminal')}
-        canRetry={false}
-      />
+      <div className="co-cloudshell-terminal__container-error">
```

Comment:
Create a new `CloudShellExec.scss` file for the styles owned by this component
```suggestion
      <div className="co-cloudshell-exec__container-error">
```